MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/model/Movies.java

```diff
@@ -0,0 +1,21 @@
+package p.lodz.pl.multiplexreservationsystem.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalTime;
+
+@Entity
+@Getter
+@Setter
+public class Movies {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private String title;
+  private LocalTime duration;
```

Comment:
Czy `LocalTime` to na pewno dobry typ dla zmiennej czasu trwania?

Reply:
Powinniśmy wykorzystywać typy najbardziej pasujące w danej sytuacji. W tym przypadku najlepszym typem byłby java.time.Duration. Kolejne przeoczenie z mojej strony ;<